syntax = "proto3";

package protos;

message HealthCheckRequest {}
message HealthCheckResponse {}
service Status {
  rpc HealthCheck(HealthCheckRequest) returns (HealthCheckResponse) {}
}
